src/inquiries.js

```diff
@@ -35,10 +35,15 @@ export const inquireHostname = isBear =>
                 when: () => isBear,
             },
             {
-                message: "Insert your hostname",
+                message: "Insert your hostname. Protocol defaults to https if none is provided.",
```

Comment:
Dose this defaulting also happen on tiger? If not please make this second sentence show only on bear.

Reply:
It does. If no protocol specified, it defaults to https. It http protocol provided on tiger, it stays the same.